modules/portal/app/views/common/footer.scala.html

```diff
@@ -57,6 +57,7 @@ <h3>@Messages("mailingList.prompt")</h3>
             <li class="@views.Helpers.maybeActive("/api")">
                 <a href="/api">API</a>
             </li>
+            @footerLink(controllers.portal.routes.Portal.externalPage("faq"), "Frequently Asked Questions")
```

Comment:
If you change the "Frequently Asked Questions" string you're providing as the second arg to `@footerLink` here to `"help.faq"` it will use the i18n system to provide translations for the other languages (sorry, I may have mislead you about the code before - you have to look in the `modules/portal/conf/messages` file to see if there is an existing code for a given piece of UI text... 

In most `.scala.html` templates you can type `Messages("some.code.here")` and it will look up the available translation. A lot of times there isn't a translation but I prefer to externalise all the UI strings anyway, i.e. put a line like `some.code.here=Some English Text` in the `messages` file. We can go over this another time in person.